### ISSUE TITLE
add support for client-side storage for meshes and textures

### ISSUE THREAD

**Issue by Foxman13** [PyriteServer/PyriteWebGL#21]

**Comment by Foxman13** [comment]
Looks like indexedDB is the best solution for this - https://hacks.mozilla.org/2012/02/storing-images-and-files-in-indexeddb/

**Comment by stefangordon** [comment]
Yeah probably the best today, but not great.  Might help on IE and Chrome.  IE will allow up to 250mb (after prompting the user for permission), which is reasonable I suppose.  Chrome will allow up to 10% of free disk space, so it is great.  Firefox and safari are really limited in space I think.

It is only valuable if the perf is actually better than reading out of the cache though, which is what is happening natively already after the first GET.  Will have to do some tests to find out.

**Comment by Foxman13** [comment]
I was going to add this at some point as an investigative feature - I could see if being beneficial with the larger meshes and the higher LOD sets where I think network would be a bottleneck.
